diff --git a/dist/index.mjs b/dist/index.mjs
index 0911f41..fd1013a 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -260,7 +260,7 @@ const generatePasskeyAuthenticationOptions = (opts, { maxAgeInSeconds }) => cre
 	const authenticationExtensions = await resolveExtensions(opts.authentication?.extensions, ctx);
 	const options = await generateAuthenticationOptions({
 		rpID: getRpID(opts, baseURLString),
-		userVerification: "preferred",
+		userVerification: "required",
 		extensions: authenticationExtensions,
 		...userPasskeys.length ? { allowCredentials: userPasskeys.map((passkey) => ({
 			id: passkey.credentialID,
@@ -324,7 +324,7 @@ const verifyPasskeyRegistration = (options) => {
 				expectedChallenge,
 				expectedOrigin: origin,
 				expectedRPID: getRpID(options, typeof ctx.context.options.baseURL === "string" ? ctx.context.options.baseURL : void 0),
-				requireUserVerification: false
+				requireUserVerification: true
 			});
 			const { verified, registrationInfo } = verification;
 			if (!verified || !registrationInfo) throw APIError.from("BAD_REQUEST", PASSKEY_ERROR_CODES.FAILED_TO_VERIFY_REGISTRATION);
@@ -425,7 +425,7 @@ const verifyPasskeyAuthentication = (options) => createAuthEndpoint("/passkey/
 				counter: passkey.counter,
 				transports: passkey.transports?.split(",")
 			},
-			requireUserVerification: false
+			requireUserVerification: true
 		});
 		const { verified } = verification;
 		if (!verified) throw APIError.from("UNAUTHORIZED", PASSKEY_ERROR_CODES.AUTHENTICATION_FAILED);
